src/extensions/default/bramble/lib/Tutorial.js

```diff
@@ -9,24 +9,43 @@ define(function (require, exports, module) {
     var Path            = Filer.Path;
 
     var PostMessageTransport = require("lib/PostMessageTransport");
-
+    var UI                   = require("lib/UI");
+    
     // Whether or not we're overriding the preview with a tutorial
     var _tutorialOverride;
 
     // When we want to override, in order to get the tutorial to
     // show, we need to force a reload for the preview.
     var _forceReload;
 
+    // Preserves the mobile view state
+    var _preserveMobileView = false;
+
     function setOverride(val) {
         _tutorialOverride = !!val;
+        
+        // If mobile view is on.
+        // we need to preserve it.
+        if ( UI.getPreviewMode() === "mobile") {
+            _preserveMobileView = true;
+        }
 
         // We need to reload the browser preview. If we're going to override,
         // we can simply reload the current live dev instance.  If not, we
         // need to properly reset live dev so it points at the project's main file.
+        
         if(_tutorialOverride) {
             _forceReload = true;
+            UI.showDesktopView(true);  // Always show in desktop mode
             PostMessageTransport.reload();
         } else {
+            
```

Comment:
I think you should consider setting the mobile restore *after* the live dev is open.  NOTE: this code is 100% untested, just gives you a guide so that things happen in the right order.

```js
} else {
  // Listen for LiveDevelopment to restart so we can switch to mobile view if necessary
  function restoreMobileView(event, status, reason) {
    if(status !== LiveDevelopment.STATUS_ACTIVE) {
      return;
    }
    LiveDevelopment.off("statusChange", restoreMobileView);
    UI.showMobileView(true);
  }
  if(_preserveMobileView) {
    _preserveMobileView = false; 
    LiveDevelopment.on("statusChange", restoreMobileView);
  }

  // If we're turning off the override, we need to open the project's
  // "main" web file, and the easiest way is to close/re-open live dev.
  LiveDevelopment.close().done(LiveDevelopment.open);
}
```